transcribed/page0010.txt

```diff
@@ -1,56 +1,56 @@
-001             TITLE ITS 1.0 3/19/67                                           PAGE 1
-002             MOBY==1
+001              TITLE ITS 1.0 3/19/67                                                 PAGE 1
+002              MOBY== 1
 003
-004             ;AC DEFS
+004              :AC DEFS
 005
-006             A=1
-007             B=1
-008             C=1
-009             D=1
-010             E=1
-011             TT=1
-012             I=1
-013             Q=1
-014             J=1
-015             R=1
-016             P=1     ;DO NOT CHANGE!
-017             T=1     ;"
-018             U=1     ;"
+006              A=1
+007              B=2
+008              C=3
+009              D=4
+010              E=5
+011              TT=6
+012              I=7
+013              Q=10
+014              J=11
+015              R=12
+016              P=15    ;DO NOT CHANGE!
+017              T=16    ;"
+018              U=17    ;"
 019
 020
-021             IOBIT==1        ;IOBIT IN MAP
+021              IOBIT==1        ;IOBIT IN MAP
 022
 023
-024             TTYIF==1        ;RIGHT HALF BITS IN PIRQC
-025             BCNTRZI==2      ;^Z TYPED
```

Comment:
Please use ^ instead of ↑.  The uparrow in 1963 ASCII has the same code as ^ in 1967 ASCII.

Reply:
aha! then I get it! that explains both ^ and _. I wasn't aware that the ASCII Standard had changed wrt to these characters!